css/styles.css

```diff
@@ -0,0 +1,346 @@
+}
+
+footer {
+  background-color: #5a5a5a;
+  align-items: center;
+  padding-top: 40px;
+  padding-bottom: 40px;
+}
+
+ul {
+  list-style: none;
+  display: inline-flex;
+  padding-left: 10%;
+}
+
+footer > ul {
+  justify-content: center;
+  width: 80%;
+}
+
+ul li {
+  padding-right: 15px;
+}
+
+footer > ul > li {
+  list-style: disc;
+  font-size: 22px;
+  margin: 10px;
+}
+
+.border-6px {
+  border-radius: 6px;
+  border: 0;
+  padding: 10px;
+}
+
+.white,
+footer,
+footer > ul > li > a,
+.last-ul > li > a,
+.profile {
+  color: white;
+}
+
+nav > a:hover {
+  background-color: rgb(48, 48, 51);
+}
+
+.more-button {
+  background-color: rgb(48, 48, 51);
+}
+
+.nav-search,
+#search-input {
+  margin-left: 0;
+}
+
+#search-input {
+  height: 80%;
+  width: 300px;
+  vertical-align: middle;
+  border-style: none;
+}
+
+#title {
+  font-size: 40px;
+  width: 480px;
+  background-color: white;
+  position: absolute;
+  left: 30%;
+  top: 18%;
+}
+
+.profile {
+  flex-direction: column;
+  width: 50%;
+  background: linear-gradient(to bottom left, lightgrey, grey);
+  height: 600px;
+}
+
+.article-line-photo img,
+.article-line-photo-small img,
+.transform-20deg {
+  transform: rotate(-20deg);
+}
+
+.profile p,
+.profile h1,
+.profile figure {
+  padding-left: 10%;
+}
+
+.article-line-photo img {
+  width: 150px;
+  height: 150px;
+  margin-right: 40px;
+  border: solid 4px black;
+}
+
+.article-line-photo-small img {
+  width: 80px;
+  height: 80px;
+  margin-right: 30px;
+  border: solid 4px black;
+}
+
+.profile figure img {
+  margin-top: 150px;
+  width: 70px;
+  height: 70px;
+}
+
+.black,
+#title {
+  color: black;
+}
+
+.grid-1,
+.grid-2,
+.article-line2,
+.grid-3,
+.last-grid {
+  display: grid;
+}
+
+.large {
+  font-size: 15px;
+}
+
+.larger {
+  font-weight: 600;
+  font-size: 20px;
+}
+
+.very-large {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 1.2;
+}
+
+.text-left {
+  text-align: left;
+}
+
+.arrow {
+  font-size: 3em;
+}
+
+.italic {
+  font-style: italic;
+}
+
+.underline {
+  text-decoration: underline;
+}
+
+.display-inline-block {
+  display: inline-block;
+}
+
+figcaption {
+  padding-left: 15px;
+}
+
+.grid-1 {
+  grid-template-columns: repeat(3, 28%);
+  margin-top: 20px;
+}
+
+.grid-2 {
+  grid-template-columns: 20% 40% 20%;
+  grid-template-rows: auto;
+  margin-top: 20px;
+}
+
+.grid-2-first {
+  background-color: #5a5a5a;
+  color: #fff;
+  border-radius: 10px;
+  padding: 40px 70px 20px 70px;
+}
+
+.grid-2-add {
+  background-color: #979797;
+  color: rgb(46, 46, 46);
+  border-radius: 10px;
+  padding: 20px 40px 20px 40px;
+}
+
+.solo-grid {
+  background-color: #5a5a5a;
+  color: #fff;
+  border-radius: 10px;
+  padding: 40px 70px 20px 70px;
+  width: 80%;
+  margin: auto;
+}
+
+.bottom-left {
+  bottom: 0;
+  left: 0;
+}
+
+.latest-article {
+  margin-top: 50px;
+  height: 150px;
+}
+
+.shipping {
+  background-color: rgb(77, 77, 77);
+  padding: 40px 23%;
+}
+
+.articles1 {
+  flex-direction: column;
```

Comment:
- [ ] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

Please apply DRY CSS to prevent repeating the same rules.